opendatahub-io/notebooks
# Move vscode and r-studio to the base layer

Move vs code, r-studio, and cuda r-studio to the base layer to be included as defaults

Ref Issue: https://github.com/opendatahub-io/notebooks/issues/283

## Description
This PR vs code, r-studio, and cuda r-studio to the base layer and removes them from the additional 

## How Has This Been Tested?
1. Deploy the operator by using the following config on the DataScienceCluster object
```
    workbenches:
      devFlags:
        manifests:
          - contextDir: components/odh-notebook-controller/config
            sourcePath: ''
            uri: 'https://github.com/opendatahub-io/kubeflow/tarball/v1.7.0-5'
          - contextDir: components/notebook-controller/config
            sourcePath: ''
            uri: 'https://github.com/opendatahub-io/kubeflow/tarball/v1.7.0-5'
          - contextDir: ''
            sourcePath: base
            uri: 'https://github.com/opendatahub-io/notebooks/tarball/pull/300/head'
      managementState: Managed
```
2. Ensure that the images are there
![image](https://github.com/opendatahub-io/notebooks/assets/42587738/784a5502-0050-42dd-9fbf-f1c29f766a11)

3. Spin up any of these three notebooks and check that they are working as expected. 

## Merge criteria:
